File: Upcoder/Khối 11/Chương 3 Cấu trúc lặp/Lãi suất ngân hàng/K11_C3_LaiSuat.cs
using System;
namespace Laixuat
{
    public class Program
    {
        static void Main(string[]args)
        {
            string [] input = Console.ReadLine().Split(' ');
            double x = double.Parse(input[0]);
            double n = double.Parse(input[1]);
            double tien = 0.0;
            if(x < 0 || n < 0)
                Console.Write("0");
            else
            {
                tien = 0;
                double temp = x;
                for(int i = (int)(n + 1); i <= 12; i++)
                {
                    tien = x + (temp * 0.05);
                    temp = x + temp * 0.05;
                }
                Console.Write(tien); 
            }
        }
    }
}

File: Upcoder/Khối 11/Chương 3 Cấu trúc lặp/Lãi suất ngân hàng/K11_C3_LaiSuat.py
x, n = map(float, input().split())
if(x < 0 or n < 0):
    print(0)
else:
    tien = 0
    temp = x
    for i in range(int(n) + 1, 12 + 1):
        tien = x + (temp * 0.05)
        temp = x + temp * 0.05
    print(tien)